airbyte-integrations/connectors/destination-azure-blob-storage/build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
 airbyteBulkConnector {
     core = 'load'
     toolkits = ['load-object-storage', 'load-azure-blob-storage']
-    cdk = 'local'
+    cdk = '0.1.19'
```

Comment:
@edgao @benmoriceau can you weigh in on how this is wrong? 

PR is failing with the following:
```
* What went wrong:
A problem occurred evaluating project ':airbyte-integrations:connectors:destination-azure-blob-storage'.
> 'cdk' should be either a well-formed version number or 'local'
```